client/src/components/overview/selectors.jsx

```diff
@@ -35,13 +34,14 @@ const Selectors = (props) => {
 
   // On size select, retrieves quantities for that size
   const retrieveQuantitiesBySize = (size) => {
+    size = size.toString();
     let maxQuantity;
 
     for (let i = 0; i < props.styles.length; i++) {
       const style = props.styles[i];
       if (style.style_id === currentStyle.style_id) {
         for (let k in style.skus) {
-          if (style.skus[k][size]) {
+          if (style.skus[k].size === size) {
```

Comment:
oof. would love to see this one refactored! any thoughts on how you break up the nested loop party here?